content/docs/instrumenting/clientlibs.md

```diff
@@ -21,6 +21,7 @@ HTTP endpoint on your application’s instance:
 Unofficial third-party client libraries:
 
 * [Bash](https://github.com/aecolley/client_bash)
+* [Clojure](https://github.com/soundcloud/prometheus-clj) (wraps the Java client library)
```

Comment:
I think we're getting crossed communication :)

> If you think a wrapper shouldn't be linked here, perhaps it would be better located where we also put gokit and such?

It's not a wrapper, nor is it a client library, nor directly instrumented code so we don't really have anywhere to put it currently. I guess a new category under 3rd party integrations for utilities?